tck/src/main/java/ee/jakarta/tck/data/framework/read/only/AsciiCharacters.java

```diff
@@ -65,7 +64,7 @@ Stream<AsciiCharacter> findByHexadecimalIgnoreCaseBetweenAndHexadecimalNotIn(Str
 
     Page<AsciiCharacter> findByNumericValueBetween(int min, int max, PageRequest<AsciiCharacter> pagination);
 
-    Streamable<AsciiCharacter> findByNumericValueLessThanEqualAndNumericValueGreaterThanEqual(int max, int min);
+//    Streamable<AsciiCharacter> findByNumericValueLessThanEqualAndNumericValueGreaterThanEqual(int max, int min);
```

Comment:
Thank you for commenting this out rather than removing it and the related tests. In a follow-on pull, I should be able to convert this into a method that returns List and then obtain multiple streams from that in order to make the tests valid again.